mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -13,36 +16,48 @@ CLASS aij net/minecraft/entity/Entity
 	FIELD O age I
 	FIELD P insideWater Z
 	FIELD Q waterHeight D
+	FIELD R wet Z
 	FIELD S inLava Z
+	FIELD T regenTimer I
+	FIELD U firstUpdate Z
 	FIELD V dataTracker Lqk;
 	FIELD W FLAGS Lqh;
 	FIELD X POSE Lqh;
+	FIELD Y updateNeeded Z
 	FIELD Z chunkX I
 	FIELD aA NAME_VISIBLE Lqh;
 	FIELD aB SILENT Lqh;
 	FIELD aC NO_GRAVITY Lqh;
 	FIELD aD invulnerable Z
 	FIELD aE scoreboardTags Ljava/util/Set;
+	FIELD aF teleportRequested Z
 	FIELD aG pistonMovementDelta [D
 	FIELD aH pistonMovementTick J
 	FIELD aI size Laik;
 	FIELD aJ standingEyeHeight F
 	FIELD aa chunkY I
 	FIELD ab chunkZ I
+	FIELD ac trackedXPosition J
```

Comment:
Maybe these could be just `trackedX`, `trackedY` and `trackedZ`? Fits better with other coordinate fields on the class